src/httphandler/login.go

```diff
@@ -0,0 +1,44 @@
+package httphandler
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/linkernetworks/logger"
+
+	"github.com/gin-contrib/sessions"
+	"github.com/gin-gonic/gin"
+)
+
+func Login(c *gin.Context) {
+
+	email := c.PostForm("email")
+	if email == "" {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"success": false,
+			"error":   fmt.Sprintf("email is required"),
+		})
+		return
+	}
+
+	password := c.PostForm("password")
+	if password == "" {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"success": false,
+			"error":   fmt.Sprintf("password is required"),
+		})
+		return
+	}
+
+	// TODO: verify email/password
+	user_id := email
```

Comment:
Can you rename `user_id` to `email` in the session? Because it's complicated to both use an ID and the email.

Reply:
OK